src/components/auth/LoginItemNewContainer.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useCallback } from 'react'
+import { ProviderType } from '@dcl/schemas/dist/dapps/provider-type'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { Loader } from 'decentraland-ui/dist/components/Loader/Loader'
+import SelectionLabel from "semantic-ui-react/dist/commonjs/elements/Label"
+import Icon from 'semantic-ui-react/dist/commonjs/elements/Icon'
+import { EthConnectAdvice } from './EthConnectAdvice'
+import './LoginItemNewContainer.css'
+
+export type LoginItemNewContainerProps = {
+  onClick?: () => void
+  onCancelLogin?: () => void
+  onCreateAccount?: () => void
+  className?: string
+  loading?: boolean
+  canceling?: boolean
+  active?: boolean
+  animate?: boolean
+  provider?: ProviderType
+  children?: React.ReactNode
+}
+
+export const LoginItemNewContainer = React.memo(
+  ({
+    children,
+    className,
+    loading,
+    canceling,
+    active,
+    provider,
+    animate,
+    onClick,
+    onCancelLogin
+  }: LoginItemNewContainerProps) => (
+    <div
+      className={`LoginItemNewContainer ${className || ''} ${loading ? 'loading' : ''}  ${animate ? 'animate' : ''} ${
+        active ? 'active' : ''
+      }`}
+      onClick={loading ? undefined : onClick}
+    >
+      {animate && <div className="LoginItemNewContainer__BackLayer" />}
+      <div className="LoginItemNewContainer__Content">{children}</div>
+      {loading && active && (
+        <div className="loader-background">
+          <Loader active={active && loading} provider={provider} size="massive" />
+          {provider && <EthConnectAdvice provider={provider} style={{ marginTop: '27px' }} />}
+          {onCancelLogin && provider && <div style={{ marginTop: '22px' }}>- or -</div>}
+          {onCancelLogin && provider && (
```

Comment:
These two seem to be having the same condition to be rendered, why don't we join them?
```suggestion
          {onCancelLogin && provider && (
          <>
            <div style={{ marginTop: '22px' }}>- or -</div>
            <Button onClick={onCancelLogin} loading={canceling} style={{ marginTop: '28px' }}>
          </>
          )}
```